react-app/src/features/selection-flow/plan-types.tsx

```diff
@@ -91,7 +91,7 @@ export const ChipSPASubmissionOptions = () => (
   <OptionsPage
     title="CHIP SPA Type"
     fieldsetLegend="Select a CHIP SPA type to create your submission"
-    options={CHIP_SPA_OPTIONS}
+    options={useChipSpaOptions()}
```

Comment:
Double check on whether this is the correct usage. Direct hook calls aren't supposed to be passed as values